torchgeo/datasets/oscd.py

```diff
@@ -406,10 +407,49 @@ def __init__(
             K.RandomCrop(patch_size), data_keys=["input", "mask"], same_on_batch=True
         )
 
+    def pad_to(
```

Comment:
I'm not sure why this can't simply reuse kornia.augmentation.PadTo and apply it to both the image and mask separately.

Reply:
@iejMac can you check if this works instead of the custom padto?

Reply:
Works great